src/sentry/logging/handlers.py

```diff
@@ -0,0 +1,29 @@
+"""
+sentry.logging.handlers
+~~~~~~~~~~~~~~~~~~~~~~~
+:copyright: (c) 2010-2016 by the Sentry Team, see AUTHORS for more details.
+:license: BSD, see LICENSE for more details.
+"""
+
+import logging
+
+
+class StructLogHandler(logging.StreamHandler):
+    def __init__(self, *args, **kwargs):
+        from structlog import get_logger
+        super(StructLogHandler, self).__init__(*args, **kwargs)
+        self._structlog = get_logger()
```

Comment:
Also, from my understanding, this relates to the `cache_logger_on_first_use` configuration setting. It's lazy until it's used once.